client/packages/programs/src/api/api.ts

```diff
@@ -196,32 +196,6 @@ export const getEncounterQueries = (sdk: Sdk, storeId: string) => ({
 
 export const getDocumentRegistryQueries = (sdk: Sdk, storeId: string) => ({
   get: {
-    byDocType: async (type: string): Promise<DocumentRegistryFragment[]> => {
-      const result = await sdk.documentRegistries({
-        filter: { documentType: { equalTo: type } },
-        storeId,
-      });
-      const entries = result?.documentRegistries;
-
-      if (entries?.__typename === 'DocumentRegistryConnector') {
-        return entries.nodes;
-      }
-      throw new Error('Error querying document registry by type');
-    },
-    byDocContext: async (
-      context: DocumentRegistryNodeContext
-    ): Promise<DocumentRegistryFragment[]> => {
-      const result = await sdk.documentRegistries({
-        filter: { context: { equalTo: context } },
-        storeId,
-      });
-      const entries = result?.documentRegistries;
-
-      if (entries?.__typename === 'DocumentRegistryConnector') {
-        return entries.nodes;
-      }
-      throw new Error('Error querying document registry by context');
-    },
```

Comment:
I found working with the more generic documentRegistries query was much easier than having these specialized queries...